### PR TITLE
Config the upgrade string index key length limit.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
NG-450

### CODE REVIEW

**Comment by critical27** [comment]
I think use 128 or 64 as default value is enough? @bright-starry-sky WDYT?

**Comment by Shylock-Hg** [comment]
> I think use 128 or 64 as default value is enough? @bright-starry-sky WDYT?

Fixed.

File: src/meta/MetaVersionMan.cpp [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 DEFINE_bool(null_type, true, "set schema to support null type");
 DEFINE_bool(print_info, false, "enable to print the rewrite data");
+DEFINE_uint32(string_index_limit, 64, "string index key length limit");
```
Comment (laura-ding) [inline src/meta/MetaVersionMan.cpp:15]: how about `index_length`, it's more simple and clear

Comment (Shylock-Hg) [reply src/meta/MetaVersionMan.cpp:15]: It's only used for string type.

**Review by jievince — APPROVED** [review]

**Review by critical27 — APPROVED** [review]

**Pull request merged**